Configure text formats form hangs

mgurjanov opened the pull request:
#720 

- [x] changes to allowed tags for full html text format

Note: The issue did come from too large allowed tags string that needs to be parsed by CKEditor and compared to already implemented tags by filters or plugins. I added tags to Full HTML text format that are not supported by already configured filters or plugins. Now text formats form response is quick and there are no any errors.

mgurjanov commented:
@mariano-dagostino Very good point and I think allowing `<form>` in WYSIWYG can pose a security issue anyway.

amitaibu commented:
@mgurjanov Can you please explain the fix, I'm not sure I understand how it's working.

mgurjanov commented:
@amitaibu `Source editing` in text format configuration form is used to manually set `allowed tags as when we use filter `Limit allowed HTML tags and correct faulty HTML` we cannot anymore manually edit `Allowed HTML tags' since core 10.2. Since that update allowed tags are defined by enabled plugins and filters. So now when we add some tags to `Source editing` section, all those entries we add, on form save get parsed and compared against enabled plugins and filters. If for example `Bold` is enabled with plugin then one cannot add `<strong>` to this section again as it's already enabled. But we can add e.g. `<div class>` so div with class is allowed etc. We shouldn't add too much of these entries anyway as any developer who builds their site should define what extra attributes should be allowed in WYSIWYG, in `Full HTML` text format or any other text format. I am not sure but I think core comes without anything in this text area.
So I removed previous entries that were too many (you can check 2 yml files in the PR) and replaced it with some that could be added as they aren't already added by enabled plugins/filters.
Also I removed `<form>` as I think it poses a security risk. Usually forms are added to site in a different way not through WYSIWYG editor.
BTW this list of filters gets read in other places in code, not just when saving this form, so too many entries affects speed of site.

Source editing where change was made|Limit allowed HTML filter
-|-
![allowed_tags](https://github.com/user-attachments/assets/844e9ee5-ff1d-4672-ae01-9c615e2f6be8)|![html_filter](https://github.com/user-attachments/assets/74d2fd37-8945-4549-b2a5-f399c7815f46)

mgurjanov commented:
@bboro Yes it has less but keep in mind that what I added in config can be seen in one yml, and the other yml holds my tags combined with other that enabled plugins/filters are adding.

@mariano-dagostino @bboro I also agree that we should probably not configure Full HTML by default.
